cfy_manager/components/prometheus/config/prometheus.yml

```diff
@@ -0,0 +1,56 @@
+# Prometheus self-check
+global:
+  scrape_interval:     15s
+  evaluation_interval: 30s
+  # scrape_timeout is set to the global default (10s).
+
+  external_labels:
+    monitor: cloudify
+
+# A scrape configuration containing exactly one endpoint to scrape:
+# Here it's Prometheus itself.
+scrape_configs:
+  - job_name: 'prometheus'
+    static_configs:
+      - targets: ['localhost:{{ prometheus.port }}']
+  - job_name: 'node'
+    static_configs:
+      - targets: ['localhost:{{ prometheus.node_exporter.metrics_port }}']
+  - job_name: 'blackbox_http_200'
+    metrics_path: /probe
+    params:
+      module: [http_200]  # Look for a HTTP 200 respose
+    static_configs:
+      - targets:
+          - http://{{ manager.private_ip }}:3000/  # Cloudify composer
+          - http://{{ manager.private_ip }}:8088/  # Cloudify stage
+          - {{ manager.external_rest_protocol }}://{{ manager.public_ip }}:{{ manager.external_rest_port }}/ # Cloudify rest and ui internal
+          - https://{{ manager.public_ip }}:53333 # Cloudify rest and ui internal
+    relabel_configs:
+      - source_labels: [__address__]
+        target_label: __param_target
+      - source_labels: [__param_target]
+        target_label: instance
+      - target_label: __address__
+        replacement: {{ manager.public_ip }}:{{ prometheus.blackbox_exporter.metrics_port }}  # The blackbox exporter's real hostname:port.
+  - job_name: 'http_400'
+    metrics_path: /probe
+    params:
+      module: [http_400]  # Look for a HTTP 400 respose
+    static_configs:
+      - targets:
+          - http://{{ manager.private_ip }}:8100/  # Cloudify rest
+          - http://{{ manager.private_ip }}:53229/  # Cloudify resources
```

Comment:
ditto private_ip

note that the same comment does NOT apply to 53333, because 53333 _is_ supposed to be exposed